src/Build.UnitTests/BackEnd/BuildManager_Tests.cs

```diff
@@ -254,8 +254,8 @@ public void SimpleGraphBuild()
             _logger.AssertLogContains("[success]");
             _logger.ProjectStartedEvents.Count.ShouldBe(1);
 
-            ProjectStartedEventArgs projectStartedEvent = _logger.ProjectStartedEvents[0];
-            Dictionary<string, string> properties = ExtractProjectStartedPropertyList(projectStartedEvent.Properties);
+            ProjectEvaluationFinishedEventArgs evalFinishedEvent = _logger.EvaluationFinishedEvents[0];
```

Comment:
Could you please clarify why we are changing the event here in the tests? 
Questions: 
- Does it mean we are moving coverage from one even to another? 
- If yes Shall we rename the `ExtractProjectStartedPropertyList` to `ExtractProjectEvaluationFinishedEvent`?

Reply:
Based on the setting of `MSBUILDLOGPROPERTIESANDITEMSAFTEREVALUATION` and the fact if loggers explicitly opt into getting properties on EvaluationFinished event - via calling in logger initialization:

```
            if (eventSource is IEventSource4 eventSource4)
            {
                eventSource4.IncludeEvaluationPropertiesAndItems();
            }
```

The properteis can be populated on the ProjectStarted or ProjectEvaluationFinished - both cases are possible, and both cases are tested in those tests.


The method should be renamed to simple `ExtractPropertyList` - good point!